src/plothist/scripts/make_examples.py

```diff
@@ -1,104 +1,191 @@
 import os
 import yaml
 import subprocess
-import argparse
-
-parser = argparse.ArgumentParser(
-    description="Redo the examples. Only run this script from the plothist/scripts folder."
-)
-parser.add_argument(
-    "--print_code", action="store_true", help="Print the code of the examples"
-)
-args = parser.parse_args()
-
-# Define folders
-plothist_folder = os.path.dirname(os.path.realpath(__file__)) + "/../../.."
-example_folder = plothist_folder + "/docs/examples"
-img_folder = plothist_folder + "/docs/img"
-temp_img_folder = plothist_folder + "/docs/temp_img"
-
-# Get all python files in the example folder
-python_files = [
-    file
-    for root, dirs, files in os.walk(example_folder)
-    for file in files
-    if file.endswith(".py")
-]
-python_files.sort()
-
-# Ask which python files to relaunch
-print(
-    "Which python script do you want to relauch? (ex: 1 2 4 ... [OR] 1d model 2d color [OR] all (just press enter for all))"
-)
-for k_python, python_file in enumerate(python_files):
-    print(f"\t{f'{k_python}':<3} - {python_file}")
-k_plots = input("> ")
-if k_plots == "":
-    k_plots = "all"
-
-# Get the list of python files to relaunch
-plots_to_redo = []
-for k_plot in k_plots.split(" "):
-    if k_plot == "all":
-        plots_to_redo = python_files[:]
-        break
-    elif k_plot in ["1d", "2d", "model", "color"]:
-        plots_to_redo.extend(
-            [
-                python_file
-                for python_file in python_files
-                if python_file.startswith(k_plot)
-            ]
+import plothist
+from pytest import fail
+import hashlib
+import matplotlib.pyplot as plt
+import warnings
+import sys
+
+# Set figure.max_open_warning to a large number to avoid warnings
+plt.rcParams["figure.max_open_warning"] = 1000
```

Comment:
Shouldn't we close the figures once they are saved to avoid the warning instead of muting it?

Reply:
Nice catch, it's an old artifact from the old method, no need now.